doc/config.rst

```diff
@@ -76,6 +76,13 @@ and will first lookup global tox settings in this section:
     * environment variable ``TOXENV``
     * ``tox.ini`` file's ``envlist``
 
+    .. versionadded:: 3.4.0
+
+        What tox environments are ran during the tox invocation can be further filtered
+        via the operating system environment variable ``TOX_SKIP_ENV`` regular expression
+        (e.g. ``py27.*`` means **don't** evaluate environments that start with the key ``py27``).
```

Comment:
is the `.` here intended?

Reply:
Yes, it is necessary. This env var is for when you want run all **but** these. ``TOXENV`` is for run these only. The ``.`` is needed and intended, because it's a regular expression, not an fnmatch. The reason for being a regular expression is to allow negation and or logic.